### ISSUE TITLE
Wrong usage of IsSet

### ISSUE THREAD

**Issue by podiki** [halogenica/beautifulhugo#67]
Thanks for the wonderful theme! Minor issue just came up:

With the release of [Hugo v0.21](https://github.com/spf13/hugo/releases/tag/v0.21) the following warning is output when using this theme:
```
WARNING: calling IsSet with unsupported type "invalid" (<nil>) will always return false.
```
This is due to a [change](https://github.com/spf13/hugo/commit/38661c17bb8c31c9f31ee18f8eba5e3bfddd5574) in `IsSet` (see [Hugo issue #3092](https://github.com/spf13/hugo/issues/3092)). I'm not sure where this is occurring exactly, and for now this is not an error (the original change to an error would break many themes, presumably including this one). However, the plan is to have this be an error in the future.

**Comment by rayjolt** [comment]
I've found the location - it's the occurrences of `{{ if isset $bigimg 0 }}` in layouts/partials/header.html. Because of the earlier use of `{{ if or $bigimg ($.Scratch.Get "title") }}`, `$bigimg` can be nil when they are rendered.

**Comment by halogenica** [comment]
I have been confused by the intention of isset (and honestly a lot of the go HTML template syntax in general), so please continue to point out whenever something like this looks wrong :) I think I fixed it with the this commit: f86398e399c828b324db76913858173acff94a1a

The warning is gone, the generated webpage appears to be correct, and it seems in line with this suggestion on the forums: https://discuss.gohugo.io/t/isset-how-to-check-variable-is-empty/1764/3
